source/Weapon.cpp

```diff
@@ -288,6 +294,11 @@ void Weapon::LoadWeapon(const DataNode &node)
 			++it;
 		}
 	}
+	
+	// Only when the weapon is not safe and has a blast radius is safeRange needed.
+	// oh, and only when it's not already overriden
```

Comment:
```suggestion
	// Only when the weapon is not safe and has a blast radius is safeRange needed, except if it is already overridden.
```
The `oh, and` feels like an oral style